tests/bdd/bdd_test.go

```diff
@@ -48,10 +48,10 @@ import (
 )
 
 var (
-	kubeconfig string
-	config     *restclient.Config
-	client     *kubernetes.Clientset
-	clientSet  *chaosClient.LitmuschaosV1alpha1Client
+	kubeconfig    string
+	config        *restclient.Config
+	client        *kubernetes.Clientset
+	dynamicClient *chaosClient.LitmuschaosV1alpha1Client
```

Comment:
Can we revert this change as this is not a dynamic client? and change the rest of the places as well